src/components/featuredContent/featuredContent.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, screen, waitFor, within } from "@testing-library/react";
+import FeaturedContentComponent from "./featuredContent";
+
+describe("Featured Content component renders with expected props", () => {
+  it("renders Digital Collections store content", async () => {
```

Comment:
The description is the same as the next test. This can be for "public domain" content.